zerver/lib/zcommand.py

```diff
@@ -1,9 +1,19 @@
 from typing import Any, Dict
 from django.utils.translation import ugettext as _
+from django.core.exceptions import ObjectDoesNotExist
 
-from zerver.models import UserProfile
-from zerver.lib.actions import do_set_user_display_setting
+from zerver.models import UserProfile, get_realm_stream, \
+    get_stream_recipient, topic_exists
+from zerver.lib.actions import do_set_user_display_setting, do_mute_topic
 from zerver.lib.exceptions import JsonableError
+from zerver.lib.actions import do_mute_topic
+from zerver.lib.topic_mutes import topic_is_muted
+from zerver.lib.bugdown import possible_linked_stream_names
+
+def get_topic_from_mute_topic_command(command: str, stream_name: str) -> str:
+    stream_link = "#**%s**" % (stream_name)
+    topic_name = command.replace('mute_topic', '').replace(stream_link, '')
```

Comment:
This will allow both `/mute_topic <topic> #<stream>` and  `/mute_topic #<stream> <topic>`.  If we decide to go with this, I'll update the usage.